templates/macros/list_posts.html

```diff
@@ -13,7 +13,7 @@
 
             <ul class="bloglist-meta">
                 {% if post.date %}
-                <li class="date">{{ macros_format_date::format_date(date=post.date, short=false, language_strings=language_strings) }}</li>
+                <li> {{ macros_format_date::format_date(date=post.updated, short=false, language_strings=language_strings) }}</li>
```

Comment:
❓ Why remove the `date` class?

I think it's better to keep the `post.date`. For example: imagine fixing a simple typo on a really old post. I think the date should reflect the timestamp of the majority of the content written, not just of the last change.

🛠️ I would consider also adding, below the original date, the `post.updated` date.

Reply:
Forgive me for my mistake.